notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -2661,6 +2661,54 @@ monster [The fire demon Surtur.]
 magic portal [Doctor Strange appeared to Thor through a portal that the doctor had conjured up.]
 
 
+movie: Beyond Skyline (2017)
+============================
+
+:: Title
+Beyond Skyline
+
+:: Description
+It is up to a ragtag band of survivors of an initial alien attack on the Earth
+to stop the aliens from completing their invasion. The film is a sequel to the
+2010 film Skyline, set concurrently with the events of the first film.
+
+:: Date
+2017-10-14
+
+:: References
+https://en.wikipedia.org/wiki/Beyond_Skyline
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+attack from outer space [The story follows a small group of people who become caught up in an alien invasion of Earth.]
+
+:: Major Themes
+father and son [The hard-boiled detective Mark Corley risked life and limb to save his son, Trent, who'd
+been taken captive by invading aliens aboard their mothership.]
```

Comment:
Incorrect line break here - don't remember if my script would be able to repair it but either way, please take care not to leave linebreaks in the motivations.

Reply:
Whoops. Fixed. While I'm still in the early stages of my investigation, we should be able to create GitHub Actions workflows to automatically catch these sorts of syntax errors.